index.bs

```diff
@@ -237,7 +237,10 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: A cryptographic device used by a [=[WAC]=] to (i) generate a [=scoped credential=] and register it with a [=[RP]=],
     and (ii) subsequently used to cryptographically sign and return, in the form of an [=Authentication Assertion=], a
     challenge and other data presented by a [=[RP]=] (in concert with the [=[WAC]=]) in order to effect
-    [=authentication=].
+    [=authentication=]. All authenticators refered to this document MUST be capable of the basic operations listed above. 
+    However, certain additive capabilites are only available among certain authenticators, such as the capability to 
+    store <a>credential private keys</a>. When an authenticator is said to be "capable of" performing a task, it means that
+    the authenticator can perform the task when requested. 
```

Comment:
Editorial nit: I am not sure that last sentence adds anything. Seems like it could be deleted.